rootfs/etc/profile.d/dietpi-unsupported_terminal.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
```

Comment:
Do we need to load the dietpi-globals for this script?
```
. /DietPi/dietpi/func/dietpi-globals
```

Reply:
@Fourdee 
Lol, was never thinking about this, but since G_* functions are used...
But I tested it several times. Seems that `/etc/bashrc.d/dietpi-login.sh` is sourced prior to `/etc/profile.d/*`. Will check to be sure.

Reply:
Jep, `/etc/profile` sources `/etc/bash.bashrc` first, which sources `/etc/bashrc.d`. `/etc/profile.d` is sourced at last, thus non of the scripts inside need to source `dietpi-globals`:
```
2018-09-01 21:40:53 root@micha:/var/log# cat /etc/profile
# /etc/profile: system-wide .profile file for the Bourne shell (sh(1))
# and Bourne compatible shells (bash(1), ksh(1), ash(1), ...).

if [ "`id -u`" -eq 0 ]; then
  PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
else
  PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games"
fi
export PATH

if [ "${PS1-}" ]; then
  if [ "${BASH-}" ] && [ "$BASH" != "/bin/sh" ]; then
    # The file bash.bashrc already sets the default PS1.
    # PS1='\h:\w\$ '
    if [ -f /etc/bash.bashrc ]; then
      . /etc/bash.bashrc
    fi
  else
    if [ "`id -u`" -eq 0 ]; then
      PS1='# '
    else
      PS1='$ '
    fi
  fi
fi

if [ -d /etc/profile.d ]; then
  for i in /etc/profile.d/*.sh; do
    if [ -r $i ]; then
      . $i
    fi
  done
  unset i
fi
```